style.css

```diff
@@ -0,0 +1,1110 @@
+.middle-image {
+  display: block;
+  margin-left: auto;
+  /* stylelint-disable-next-line declaration-block-no-duplicate-properties */
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.middle-image1 {
+  display: none;
+}
+
+.middle-1-container {
+  margin-left: 43px;
+  margin-right: 24px;
+}
+
+.middle-1-container h1 {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  margin-left: 14px;
+}
+
+.middle-1-container button {
+  flex-wrap: wrap;
+  width: auto;
+  height: 40px;
+  margin-block-end: 10px;
+  align-items: center;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  margin-right: 12px;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  color: black;
+  font-family: Inter, sans-serif;
+  margin-left: 10px;
+  list-style: none;
+}
+
+#see-button-one {
+  height: 48px;
+  background: #ff6b00;
+  font-family: Inter, sans-serif;
+  color: white;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.middle-1-container p {
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 10px;
+}
+
+.middle-1-container1 {
+  background-image: url("./watermarker.png");
+  width: 327px;
+  height: 386px;
+  margin-top: 37px;
+  margin-left: auto;
+  margin-right: auto;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+  margin-bottom: 24px;
+}
+
+.middle-1-container1 h1 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+  margin-left: 16px;
+  margin-right: 16px;
+  padding-top: 84px;
+}
+
+#skill {
+  margin-top: 84px;
+}
+
+.middle-1-container1 p {
+  width: 295px;
+  height: 87px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 12px;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+  margin-bottom: 12px;
+}
+
+.middle-1-container1-btn {
+  width: auto;
+  height: 40px;
+  background: rgb(255 255 255 / 24%);
+  margin-left: 16px;
+  margin-right: 14px;
+  margin-bottom: 22px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.see-button-one1 {
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  margin: 0%;
+  border: #ff6b00;
+}
+
+.about-container {
+  background-color: black;
+  background-image: url("aboutmepic.png");
+  background-repeat: no-repeat;
+  background-position: center right;
+  background-position-y: 350px;
+}
+
+.about-discription-container {
+  padding-bottom: 142px;
+}
+
+.about h1 {

```

Comment:
- [ ] Please, try to fix the alignment of icons in this section, as previous reviewer mentioned:
<img width="1222" alt="Screen Shot 2022-03-31 at 11 54 19" src="https://user-images.githubusercontent.com/49064106/161008089-5b702c4a-0d31-4566-b8bb-ba61807dbd42.png">